received_friend.py

```diff
@@ -0,0 +1,153 @@
+import matplotlib.pyplot as plt
+import os
+import timestring
+import datetime
+from datetime import timedelta
+import dateutil.parser
+import numpy as np
+import json
+from tabulate import tabulate
+
+
+
+today = datetime.datetime.now().strftime('%b %d, %y')
+yesterday = (datetime.datetime.now() - timedelta(days=1)).strftime('%b %d, %y')
+
+
+
+def friends():
+    loc = input('Enter facebook archive extracted location: ')
+    if not os.path.isdir(loc):
+        print("The provided location doesn't seem to be right")
+        exit(1)
+    
+    rname = loc+'/friends/received_friend_requests.json'
```

Comment:
these are constants so it will be good if it's moved up and declared in the beginning of the file.